build.psm1

```diff
@@ -453,6 +454,13 @@ Fix steps:
         $Arguments += "/property:ReleaseTag=$ReleaseTagToUse"
     }
 
+    if (-not $SkipRoslynAnalyzers) {
+        $Arguments += "/property:RunAnalyzersDuringBuild=true"
+    }
+    else {
+        $Arguments += "/property:RunAnalyzersDuringBuild=false"
+    }
+
```

Comment:
```suggestion
    if ($SkipRoslynAnalyzers) {
        $Arguments += "/property:RunAnalyzersDuringBuild=false"
    }
    else {
        $Arguments += "/property:RunAnalyzersDuringBuild=true"
    }

```

Reply:
I removed `RunAnalyzersDuringBuild=true` altogether as this could override setting in csproj.